src/reviewer.js

```diff
@@ -34,6 +34,15 @@ function fetch_other_group_members({ author, config }) {
 }
 
 function identify_reviewers_by_changed_files({ config, changed_files, excludes = [] }) {
+  const DEFAULT_OPTIONS = {
+    last_match_only: false,
```

Comment:
I'm now wondering if this option is clear enough since this only works for `files` 🤔 Let me check the whole configuration.

Reply:
Yeah, it can be confusing. This action can be used without `files`, for instance.
Just brainstorming now...

- `last_files_match_only`
- `last_matching_files_pattern_only`
- `last_matching_files_only`

Reply:
Cool, I'll go with `last_files_match_only`.

Reply:
I agree it's better. Thank you!